clr-k8s-examples/tests/cpumanager/test-cpumanager.yaml.tmpl

```diff
@@ -0,0 +1,92 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-cpumanager-guaranteed-__runtimeclass__
+spec:
+  restartPolicy: Never
+  containers:
+  - name: busy
+    image: busybox
+    command: [ "top" ]
+    resources:
+      limits:
+        cpu: 1
+        memory: 100Mi
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-cpumanager-burstable-integer-limit-__runtimeclass__
+spec:
+  restartPolicy: Never
+  containers:
+  - name: busy
+    image: busybox
+    command: [ "top" ]
+    resources:
```

Comment:
Would it be useful to have multi-container pods here?  Can help test placement inside the pod as well as on host.

Reply:
This becomes relevant when you use Kata...